Tests/TestData/SelfVerifyingTests/VBToCS/InheritanceTests.vb

```diff
@@ -36,24 +48,27 @@ Module Program
             Dim nonVirtualMethodGood2 = Me.F2() = 2
             Dim sharedMethodGood1 = MyClass.F3() = 3
             Dim sharedMethodGood2 = A.F3() = 3
+            Dim interfaceMethodGood = CType(Me, IA).F4() = 11
```

Comment:
I notice that the *source* [test failed](https://github.com/icsharpcode/CodeConverter/runs/4980385369?check_suite_focus=true#step:9:29) (i.e. even before it was converted)
Presumably this is because F4 is overridden to be 44 and this expected it to be 11

Reply:
Ohh, right. I was playing too much with it :D It still fails but on the converted code.

Reply:
Yep looks good now